### DISCUSSION TITLE
Github Readme doesn't load markdown well

### DISCUSSION THREAD

**Discussion by bayurzxsmtp** [community/community#60632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Feature Area

Issues

### Body

I noticed that html properties do not load well in the github readme but works well on vscode plugin extension(Markdown Preview) and some other sites. I made some screenshots about it below
## Code:
![image](https://github.com/community/community/assets/92230919/2e46fbb9-0158-4695-9a0d-0ea4d2b01148)

## What it looks like
![image](https://github.com/community/community/assets/92230919/d64d2241-53d4-48a1-8eab-73efdc6110d3)

## What it should look like
![image](https://github.com/community/community/assets/92230919/4876e810-23ee-450f-aa55-cb6aec6d020c)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bayurzxsmtp ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by zm-cttae** [comment]
These are raw HTML and also would cause serious A11Y issues (the MySpace effect) if they were allowed out of images.

**Comment by fedorkanan** [comment]
hello @bayurzxsmtp 
nice to tall to you <3 
because these are raw html code and can`t generate your code in your profile you should use specific type of code for gitHUB not the raw HTML tags 
use this website to generate a good looking GITHUB read.me :

https://rahuldkjain.github.io/gh-profile-readme-generator/

https://readme.so/

https://gprm.itsvg.in/

https://profilinator.rishav.dev/

this are the 5 best website for making GIT hub read.me and can gave you the code you need
